UnityProject/Assets/Scripts/UI/ProgressBar/ProgressBar.cs

```diff
@@ -255,7 +255,7 @@ void UpdateMe()
 	public void ServerInterruptProgress()
 	{
 		//already closed?
-		if (done) return;
+		if (done || progressAction == null) return;
```

Comment:
NullReferenceException: Object reference not set to an instance of an object
ProgressBar.ServerInterruptProgress () (at /github/workspace/UnityProject/Assets/Scripts/UI/ProgressBar/ProgressBar.cs:260)
StandardProgressAction.InterruptProgress (System.String reason) (at /github/workspace/UnityProject/Assets/Scripts/UI/ProgressBar/StandardProgressAction.cs:253)
StandardProgressAction.OnLocalPositionChanged (UnityEngine.Vector3Int arg0) (at /github/workspace/UnityProject/Assets/Scripts/UI/ProgressBar/StandardProgressAction.cs:294)
UnityEngine.Events.InvokableCall`1[T1].Invoke (T1 args0) (at /home/builduser/buildslave/unity/build/Runtime/Export/UnityEvent/UnityEvent.cs:207)
UnityEngine.Events.UnityEvent`1[T0].Invoke (T0 arg0) (at /home/builduser/buildslave/unity/build/Runtime/Export/UnityEvent/UnityEvent/UnityEvent_1.cs:58)
RegisterTile.UpdatePositionServer () (at /github/workspace/UnityProject/Assets/Scripts/Tilemaps/Behaviours/Objects/RegisterTile.cs:492)
PlayerSync.Synchronize () (at /github/workspace/UnityProject/Assets/Scripts/Player/PlayerSync.cs:500)
PlayerSync.UpdateMe () (at /github/workspace/UnityProject/Assets/Scripts/Player/PlayerSync.cs:444)
UpdateManager.Update () (at /github/workspace/UnityProject/Assets/Scripts/Managers/UpdateManager/UpdateManager.cs:253)